source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -47,36 +47,30 @@ Installing the MinIO :ref:`Kubernetes Operator <deploy-operator-kubernetes>` aut
 This documentation assumes familiarity with all referenced Kubernetes concepts, utilities, and procedures. 
 While this documentation *may* provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kubernetes Documentation <>`.
 
-.. _deploy-minio-distributed-prereqs-storage:
+
+
+
+
 
 Prerequisites
 -------------
 
-MinIO Kubernetes Operator and Plugin
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+MinIO Kubernetes Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The procedures on this page *requires* a valid installation of the MinIO
 Kubernetes Operator and assumes the local host has a matching installation of
-the MinIO Kubernetes Operator. This procedure assumes the latest stable Operator
-and Plugin version |operator-version-stable|.
+the MinIO Kubernetes Operator. This procedure assumes the latest stable Operator, version |operator-version-stable|.
 
 See :ref:`deploy-operator-kubernetes` for complete documentation on deploying the MinIO Operator.
 
-.. cond:: k8s and not openshift
-
-   .. include:: /includes/k8s/install-minio-kubectl-plugin.rst
-
-.. cond:: openshift
-
-   .. include:: /includes/openshift/install-minio-kubectl-plugin.rst
-
 .. cond:: k8s and not (openshift or eks or gke or aks)
 
-   Kubernetes Version 1.19.0
+   Kubernetes Version 1.21.0
    ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   Starting with v4.0.0, the MinIO Operator requires Kubernetes 1.19.0 and later.
-   The Kubernetes infrastructure *and* the ``kubectl`` CLI tool must have the same version of 1.19.0+.
+   MinIO Operator requires Kubernetes 1.21.0 and later.
```

Comment:
Not sure how you have all versions of Kubernetes from 1.21.0 on installed at the same time. 😆 

```suggestion
   MinIO Operator requires Kubernetes 1.21.0 or later.
```

Reply:
_ALL THE VERSIONS!!_ 🤣